### PR TITLE
feat(postgres): create postgres container for production

### PR DESCRIPTION
**Pull request by luabida**
Start working in postgres container for production (with postgis).

image: `postgres:15-alpine3.17`

### CODE REVIEW

**Comment by luabida** [comment]
@xmnlab  I've bind `/var/lib/postgresql/data` to $PGDATA on container and got: 
```
2023-05-05 21:51:47.751 GMT [1] LOG:  skipping missing configuration file "/var/lib/postgresql/data/pgdata/postgresql.auto.conf"
2023-05-05 21:51:47.751 GMT [1] FATAL:  data directory "/var/lib/postgresql/data/pgdata" has wrong ownership
2023-05-05 21:51:47.751 GMT [1] HINT:  The server must be started by the user that owns the data directory.
```

Any idea why it could happen?

File: containers/postgresql/docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+
+if [ "$ENV" == "dev" ]; then
+  # prepare-db
```
Comment (luabida) [inline containers/postgresql/docker-entrypoint.sh:6]: @xmnlab There is dev scripts being copied to `/docker-entrypoint-initdb.d/` but they don't seem to initialize automatically, what you thing would be the best approach to prepare dev db?

File: containers/postgresql/docker-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+fi
+
+exec postgres "$@"
```
Comment (luabida) [inline containers/postgresql/docker-entrypoint.sh:9]: @xmnlab without `postgres` here, the entrypoint was breaking the container with the log: `/docker-entrypoint.sh: line 7: exec: -p: invalid option`. Any idea?

**Comment by luabida** [comment]
Dev postgres container is not running scripts located at the mount `/docker-entrypoint-initdb.d/`, probably because they don't have execute permission when the volume is binded with compose-dev

File: containers/postgresql/scripts/dev/prepare-db.sh
Comment (luabida) [inline containers/postgresql/scripts/dev/prepare-db.sh:1]: @xmnlab how to create a file to initialize the dev db as this make command was doing?

**Comment by fccoelho** [comment]
We should integrate in this PR moving the data directory to the volume in the /mnt, otherwise we risk running out of disk space in the host soon

**Review by xmnlab — APPROVED** [review]
it looks good to me.
maybe before applying this change to the production we could test it on the staging server (kobo)

**Pull request merged**